### PR TITLE
Update packages.json

### PR DESCRIPTION
**Pull request by qianhongqiang**
add HQMentionBlock

### CODE REVIEW

**Comment by jurre** [comment]
The JSON file is split up into 4 different parts: `plugins`, `file_templates`, `color_schemes` and `project_templates`. You've added this as a project template, it should go under the `plugins` key.

File: packages.json [hunk 1]
```diff
@@ -2320,6 +2320,12 @@
         "url": "https://github.com/Isuru-Nanayakkara/Xcode-Empty-Application-Template",
         "description": "Bringing the Empty Application template back to Xcode",
         "screenshot": "https://raw.githubusercontent.com/Isuru-Nanayakkara/Xcode-Empty-Application-Template/master/screenshot.png"
+      },
+ +    {
+ +      "name": "HQBlockMention",
+ +      "url": "https://github.com/qianhongqiang/HQBlockMention.git",
+ +      "description": "A plugin for Xcode,which will remind you retain circle in block.",
+ +      "screenshot": "https://github.com/qianhongqiang/HQBlockMention/blob/master/HQBlockMentionScreenSHot.png"
```
Comment (jurre) [inline packages.json]: I think you copied the `+`s from the diff making the JSON invalid

Comment (qianhongqiang) [inline packages.json]: I'm sorry.

**Comment by qianhongqiang** [comment]
what should I do else?is this commit ok?

**Comment by jurre** [comment]
The plugin is still not listed under the `plugins` key, the install won't work this way.

**Pull request closed without merging**